### PR TITLE
Fixes #99

### LINKED ISSUE #99: Add new Section for EAA Exceptions
We need a new section for EAA Exceptions.

75 EAA exception 1 – Micro-enterprises
76 EAA exception 2 – Disproportionate burden
77 EAA exception 3 – Fundamental modification

@mattgarrish you recommended

| epub | onix |
| ----- | ----- |
| a11y:exemption with the value "microenterprise" | Code 75: EAA exception 1 - Micro-enterprises |

Where should this go in our current document?

Would it go under 1. EPUB and ONIX

Or a new section 3. EAA Exceptions ?

### PR DESCRIPTION
**Pull request by clapierre**
Added all the new EAA exceptions to the bottom of Section 1.0 EPUB / ONIX

### CODE REVIEW

**Review by madeleinerothberg — APPROVED** [review]

File: crosswalk/index.html [hunk 1]
```diff
@@ -276,6 +276,31 @@ <h2>EPUB and ONIX</h2>
 								Specification 1.0 AA</p>
 						</td>
 					</tr>
+                    
+					<tr>
+						<td>
+							<p><code>a11y:exemption</code> with the text string<br/><code>microenterprise</code></p>
```
Comment (mattgarrish) [inline crosswalk/index.html:282]: The values look like the ONIX descriptions. The [three epub codes](https://www.w3.org/TR/epub-a11y-exemption/#exemption-values) you use with the property are:

- eaa-microenterprise
- eaa-disproportionate-burden
- eaa-fundamental-alteration

**Review by mattgarrish — CHANGES_REQUESTED** [review]

**Comment by clapierre** [comment]
Matt I like this, but wouldn't the last one be
for
EAA exception 3 - Fundamental modification

be: 
eaa-fundamental-modification
instead of
eaa-fundamental-alteration
to be in line with what ONIX has?

**Comment by mattgarrish** [comment]
> eaa-fundamental-modification

That's not the language of the EAA. It's quoted in the property value definition I linked (emphasis mine):

> shall apply only to the extent that compliance: (a) does not require a significant change in a product or service that results in the **_fundamental alteration_** of its basic nature

I'm not sure why ONIX uses a different word in the description of the code number, but since it's only a description it's not quite the same as having the value in the metadata.

**Comment by clapierre** [comment]
All set @mattgarrish, @gregoriopellegrino and @madeleinerothberg we good to go?

**Review by mattgarrish — APPROVED** [review]

**Pull request merged**